src/components/map-marker/map-marker.tsx

```diff
@@ -0,0 +1,118 @@
+import { Component, Element, Event, EventEmitter, Prop } from "@stencil/core";
+import { IComponent } from "../common/interfaces";
+import {
+  icon,
+  marker
+  // tslint:disable-next-line:no-submodule-imports
+} from "leaflet/dist/leaflet-src.esm";
+
+@Component({
+  shadow: false,
+  styleUrl: "map-marker.scss",
+  tag: "gx-map-marker"
+})
+export class MapMarker implements IComponent {
+  @Element() element: HTMLElement;
+
+  private markerInstance: any;
+
+  /**
+   * The coordinates where the marker will appear in the map.
+   *
+   */
+  @Prop({ mutable: true })
+  coords = "0, 0";
+
+  /**
+   * The src of marker img. It can be an URL or a file path.
+   *
+   */
+  @Prop({ mutable: true })
+  iconSrc = "https://unpkg.com/leaflet@1.3.4/dist/images/marker-icon.png";
```

Comment:
Isn't this icon included in the package distribution? If so, we could copy it as a library asset so we don't need to rely on a third party.

Reply:
Ready. Using an encrypted img. Don't need to rely on a third party, and it's not possible load from package distribution.